### ISSUE TITLE
AutoNumeric - RTL (right-to-left script)

### ISSUE THREAD

**Issue by jarciano** [autoNumeric/autoNumeric#694]
### Current behavior
The user has to delete the zeros "**R$ 0,00**" and after that type **1,23** manually inserted the comma "," to have the "**R$ 1,23**" because if you type 123 automatically it becomes "**R$ 123,00**"

Another detail if positioning the cursor on the right simply does nothing when trying to type any number "**R$ 0,00|**"

### Expected behavior
I need to format it with monetary value where the user enters the numbers from right to left "RTL" (right-to-left script).

Example: the cursor starts at the end of the numbers "**R$ 0,00|**" and when you enter the value "**123**" it moves from right to left "**R$ 1,23**"

### Steps to reproduce the problem
1. Use autoNumeric version 4.6.0 (Primefaces InputNumber uses AutoNumeric under the covers)
2. In the browser Chrome: 84 | Firefox: 78
3. On the Linux and Windows operating system

### Link to live example (https://www.dropbox.com/s/dr2g38rq7hl9g6i/inputNumber.mp4?dl=0)

Watch this example video from another application, the cursor starts at the end of the numbers R$ 0,00| and when entering the value 123 it moves from right to left R$ 1,23

### Detail
I am opening this issue after trying a solution in the Primefaces Forum @melloware told me to try this feature with you.

https://forum.primefaces.org/viewtopic.php?p=183472#p183472

Thanks attention, and congratulations on the work

**Comment by jarciano** [comment]
Hello,
@AlexandreBonneau i imagine it is full of work, but any good news about my request for improvement?

**Comment by jarciano** [comment]
Hello,
@AlexandreBonneau i imagine it is full of work, but any good news about my request for improvement?

**Comment by jarciano** [comment]
Hello,

It's been a few years and I haven't seen any news about the subject, @AlexandreBonneau would you have any plans for such a function?

**Comment by vitor-fontoura** [comment]
One and a half year and still no feature to do it. This is the closest i could get with the current options:
```
{
    showOnlyNumbersOnFocus: true, // If you dont do this, it gets really confusing to the end user
    divisorWhenUnfocused: 100, // Dividing here would mean that inputing 123 would result in 1,23. Be aware that it depends on your decimalPlaces
    decimalPlacesShownOnFocus: 0, // To hide the ',00' after the 123 when focusing on the field
    rawValueDivisor: 100 // Else it would be an integer
}
```
Hope it can help while the RTL feature is missing, and if someone has a better alternative using the existing options please share with us 🙏

**Comment by melloware** [comment]
@vitor-fontoura i think you will either have to provide a PR or "financially sponsor" this feature with @AlexandreBonneau ...

**Comment by vitor-fontoura** [comment]
> @vitor-fontoura i think you will either have to provide a PR or "financially sponsor" this feature with @AlexandreBonneau ...

Fair enough. My intention was more to share the workaround with others like me that happen to find this thread looking for a feature like this.

**Comment by AlexandreBonneau** [comment]
Indeed a PR is welcome, since this touch the basis of AutoNumeric.
Please describe more the behavior you want to see ;
for instance when you have `1234|5,67`,
when you enter `8`, should you get:
a/ `123485,67`
b/ `12385,67`
